opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -144,20 +149,29 @@ def from_map(cls, maxlen, mapping):
         return bounded_dict
 
 
-class SpanContext(trace_api.SpanContext):
-    """See `opentelemetry.trace.SpanContext`."""
-
-    def is_valid(self) -> bool:
-        return (self.trace_id == trace_api.INVALID_TRACE_ID or
-                self.span_id == trace_api.INVALID_SPAN_ID)
-
-
 Event = namedtuple('Event', ('name', 'attributes'))
 
 Link = namedtuple('Link', ('context', 'attributes'))
 
 
 class Span(trace_api.Span):
+    """See `opentelemetry.trace.Span`.
+
+    Users should generally create `Span`s via the `Tracer` instead of this
```

Comment:
I think we should highlight this in a stronger tone - I'd really disliked users creating `Span`s directly (unless you guys have a specific reason/case in mind).

Reply:
No specific use case, how about removing "generally"?